fitlib/objects.py

```diff
@@ -33,7 +34,9 @@ def getUpstreamItems():
 
 def getDownstreamItems(fitTrackedData, paths, stats):
     cached = cache.find((fitTrackedData[p][0] for p in paths), update=False)
-    return [p for p in paths if not (p in stats or fitTrackedData[p][0] in cached)]
+    skippedFiles = []
```

Comment:
The files in skippedFiles[] won't be downloaded on an individual user machine, but will still be tracked by git-fit, and will be available in the remote storage. Also, user can download any skipped files any time - just got to remove the particular extension(s) from skip list.